Gopkg.toml

```diff
@@ -76,3 +76,7 @@
 [[prune.project]]
   name = "github.com/kubernetes-csi/external-snapshotter"
   non-go = false
+
+[[constraint]]
```

Comment:
@nixpanic not sure whats the plan in go ceph project about the releases. It would be really better if we can stick onto the release versions than the `master`.

Reply:
yes, it's better to use the release version

Reply:
There is no timeline for a release yet. Some APIs might still change (unlikely for functions that are used now).

Reply:
If the apis which we are consuming here are respected in future releases, its fine to stick on to `master` for now.